src/functionalTest/resources/scenarios/RIA-2527-resident-judge-record-appellant-ftpa-decision-new.json

```diff
@@ -0,0 +1,90 @@
+{
+  "description": "RIA-2527 Decide FT{A application appellant",
+  "launchDarklyKey": "dlrm-setaside-feature-flag:true",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 1234,
+      "eventId": "decideFtpaApplication",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ftpaApplicantType": "appellant",
+          "ftpaAppellantSubmitted": "Yes",
+          "ftpaAppellantRjDecisionOutcomeType": "granted",
+          "ariaListingReference": "987654321",
+          "listCaseHearingCentre": "taylorHouse",
+          "ftpaAppellantDecisionDocument": [
```

Comment:
This should be ftpaApplicationApellantDocument but I can change it at my ticket https://tools.hmcts.net/jira/browse/RIA-8592 since I have to review the notifications